src/main/java/sammy/Sammy.java

```diff
@@ -0,0 +1,45 @@
+package sammy;
+
+import java.io.IOException;
+public class Sammy {
+
+    private Storage storage;
+    private TaskList tasks;
+    private Ui ui;
+
+    public Sammy(String filePath) {
+        ui = new Ui();
+        storage = new Storage(filePath);
+        try {
+            tasks = new TaskList(storage.load());
+        } catch (IOException e) {
+            ui.showErrorMessage("Error loading tasks from file");
+            tasks = new TaskList();
+        }
+    }
+
+    public void run() {
+        ui.showWelcomeMessage();
+        boolean isExit = false;
```

Comment:
Unlike the isExit boolean in the Command class which is easily understood, the isExit boolean here does not have the context of being related to a command. Would be helpful if the wording could be change to something along the lines of 'isExitCommandReceived()'